### PR TITLE
Update docs related to services catalog

### PR DESCRIPTION
**Pull request by Waseem826**

### CODE REVIEW

File: content/developer-platform/platform-users/consuming-services/service-catalog.jpg
Comment (embik) [inline content/developer-platform/platform-users/consuming-services/service-catalog.jpg:1]: Can you maybe take this screenshot on the demo environment?  This one has a lot of clutter from our tests on dev, and I'd prefer to not show e.g. the "Matrix" thing on our official docs.

I also wonder if it should be cropped a little to show it's a popup, but not the whole page.

Comment (Waseem826) [reply content/developer-platform/platform-users/consuming-services/service-catalog.jpg:1]: We don't have this latest UI on the demo environment but I get your point and I'll update the screenshot.

Comment (Waseem826) [reply content/developer-platform/platform-users/consuming-services/service-catalog.jpg:1]: Updated!

Comment (embik) [reply content/developer-platform/platform-users/consuming-services/service-catalog.jpg:1]: True, sorry! I didn't think that far. My bad. Thanks for getting it updated anyway!

**Review by embik — APPROVED** [review]
/approve

thanks!

**Pull request merged**